database/buildSchema/28_review_question_assignment.sql

```diff
@@ -5,3 +5,31 @@ CREATE TABLE public.review_question_assignment (
 	template_element_id integer references public.template_element(id),
 	review_assignment_id integer references public.review_assignment(id)
 );
+
+
+-- Function to return count of assigned questions for current stage/level
+CREATE FUNCTION public.assigned_questions_count(curr_ra public.review_assignment)
```

Comment:
Hmm. I still think it's better to have this method to accept the following params:
`application`, `stage_id`, `level`. 

It would avoid extra sub queries. And should look neater.

Reply:
Cool, have made suggested changes, seems to be working well now.

I've returned the `is_fully_assigned` column as `is_fully_assigned_level_1` -- you think this is okay?